### DISCUSSION TITLE
Knowledge Retrieval Problems

### DISCUSSION THREAD

**Discussion by FelGi671** [langgenius/dify#11564] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Hello there!

I just get started with Dify. I added LLMs like llama and Qwen and also a few different embedding models. For my first chatbot i added a knowledge retrieval. The problem is that the chatbot is unable to get the information out of that. 
For now i have no rerank model integrated. This setting is also not active. 
Anybody have or had the same problem and can help me out? I have no clue what to do. 

Thanks for ur help.
![image001](https://github.com/user-attachments/assets/3023f9d8-3a20-44f3-a267-4504b00db944)
![image001](https://github.com/user-attachments/assets/a310bcf4-0c08-4126-9a3b-dd20d3e6d612)
![image](https://github.com/user-attachments/assets/0dea4321-bf02-4bd3-85b6-bf8f658d035e)

**Comment by larkinmd7** [comment]
go into knowledge base test mode, write something there and show me if you have an error there? 
![photo_2024-12-12 10 21 49](https://github.com/user-attachments/assets/915c7f62-ab02-41cd-92d7-91e8e4ad79ed)
![photo_2024-12-12 10 20 54](https://github.com/user-attachments/assets/381f29c7-a7d5-4d4a-a0fd-2fce1ecd9080)

I'm having errors licked, I don't understand why that's happening.
